### ISSUE TITLE
Trading212 anomalies in import

### ISSUE THREAD

**Issue by vajny** [dickwolff/Export-To-Ghostfolio#133]
Hi,
I found some anomaly on Trading 212 when the currency of share price is reported as GBX (in my case) the converted output is USD and does not make any sense.

![image](https://github.com/user-attachments/assets/50f880c6-a10a-442a-824d-9c81857ff30c)

![image](https://github.com/user-attachments/assets/9b17fdb6-80f2-4f2b-8fda-e52c0ec9a9d9)

![image](https://github.com/user-attachments/assets/179dc97f-a45e-4a58-bed6-bb056a705af8)

So when I import it to ghostfolio it then makes this nonsense:
![image](https://github.com/user-attachments/assets/101787cb-0c17-4f5d-b6b0-5df880a606a8)

Hope the screenshots talks for themselves. If needed I will describe the issue more or send the files.

Small FYI, because I had to google it myself.
1 GBX = 0.01 GBP
It is some pseudo currency in use for London Stock Exchange.

**Comment by dickwolff** [comment]
Thanks for reporting this, and sorry for the late response. I’ve been busy finishing things up last year and now I have more time to look into issues for this project.

I have previously added something to convert GBX to GBp. If I understand your description here correctly, the output of this tool makes the transaction a USD instead of GBp?

**Comment by dickwolff** [comment]
In one of the most recent releases I reverted a change regarding symbol mapping. @vajny Can you try again with the latest version and see if the issue is still there?

**Comment by vajny** [comment]
Hi,
looks like now its correctly reporting as GBp to Ghostfolio, but it cannot handle the historical price. Since it is not a real currency.

I would say, that the problem is for ghostfolio to fix now.

Or would you be able to make a simple conversion for GBp(GBX) to GBP since it is just price*0.1 inside of the convert script?

That would solve the problem forever. It is occurring on every portfolio manager I have tried, so you would probably be the first person to fix this issue.


![image](https://github.com/user-attachments/assets/276e8f48-c8b1-4e24-aa75-9665d9075285)

**Comment by dickwolff** [comment]
I'll take a look at it and let you know!

**Comment by dickwolff** [comment]
@vajny I added something to map from GBp to GBP. Can you verify if it's working for you by running [this branch](https://github.com/dickwolff/Export-To-Ghostfolio/tree/feat/T212-Gbx-To-GBP) or [this Docker image](https://hub.docker.com/layers/dickwolff/export-to-ghostfolio/0.22.3-PullRequest0157.4/images/sha256-d95735ec666e34572ace6cd0ad10bb96c5f8be31a103bc302e84710bf488b4cb)? Thanks!

**Comment by vajny** [comment]
Hi, 
It gives me another error now. 
![image](https://github.com/user-attachments/assets/d44c30f3-a849-408b-8bdc-9f8c0e42c8e7)
I can send you the file I am converting if it helps you.

**Comment by dickwolff** [comment]
Yeah this is the problem with overriding the GBP, because it's GBp on Yahoo Finance and then Ghostfolio expects the record to match that. 😞

**Comment by vajny** [comment]
Yeah, we are running in a circle. Maybe it is just a matter of correctly setting the exchange rate manually in ghostfolio. But I was not successful with that. I really appreciate your attempt to help, but unfortunately it looks like I will have to return to my old excel sheets instead of ghostfolio.

**Comment by dickwolff** [comment]
It’s unfortunate. Closing from my side.

Maybe @dtslvr can point you to the right direction on how to set up the exchange rate?
